index.htm

```diff
@@ -0,0 +1,101 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>hack next</title>
+  <meta name="description" content="a friendly cabin hack in mid-sweden, bring your friends!">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="main.css">
+
+  <!-- social media share metadata -->
+  <meta name="twitter:card" content="summary" />
+  <meta name="twitter:site" content="@stefan_wallin" />
+  <meta property="og:url" content="https://hacknextse.github.io/" />
+  <meta property="og:title" content="hack next" />
+  <meta property="og:type" content="website" />
+  <meta property="og:description" content="a friendly cabin hack in mid-sweden, bring your friends!" />
+  <meta property="og:image" content="https://hacknextse.github.io/social-card.png" />
+  <!-- Favicons -->
+  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+  <link rel="manifest" href="/site.webmanifest">
+  <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#282d34">
+  <meta name="msapplication-TileColor" content="#282d34">
+  <meta name="theme-color" content="#282D34">
+
+</head>
+
+<body>
+  <header>
+    <img src="logo.png" width="200px" alt="hack next logo"/>
+  </header>
+
+  <main>
+    <h2>what</h2>
+    <p>a cabin-hack in the middle of Sweden</p>
+    <p>since 2014 a small group of developers has gotten together in a cabin in Järvsö to hack on hobby projects, and eat and drink well — now we want more hackers!
```

Comment:
```suggestion
    <p>since 2014 a small self-organizing hack have taken place in Järvsö, groups of friends have meet to hack on hobby projects, socialize and eat well !
```